server/routes/handlers/helpers/dates.ts

```diff
@@ -137,15 +137,62 @@ export const formatDateTimeFromIsoString = (isoDate: string) => {
   }
   try {
     const includeTime = isoDate.length > 10
-    const date = DateTime.fromISO(isoDate, { zone: 'utc' }).setZone(timeZone)
+    const dateTime = getDateTimeInEuropeLondon(isoDate)
 
     if (includeTime) {
-      return date.toFormat("d MMMM yyyy' at 'HH:mm")
+      return dateTime.toFormat("d MMMM yyyy' at 'HH:mm")
     }
-    return date.toFormat('d MMMM yyyy')
+    return dateTime.toFormat('d MMMM yyyy')
   } catch (err) {
     return ''
   }
 }
 
 export const dateHasError = (field: string | DateValidationError) => Boolean((field as DateValidationError).error)
+
+export const recallAssessmentDueText = (isoDate?: string): string | undefined => {
```

Comment:
Why is isoDate param optional, wouldn't we want a TS error if the function was called without it?

Reply:
Fixed.